plugins/outputs/cloudwatch/cloudwatch.go

```diff
@@ -398,6 +399,10 @@ func (c *CloudWatch) BuildMetricDatum(metric *aggregationDatum) []*cloudwatch.Me
 			continue
 		}
 		if len(distList) == 0 {
+			if math.IsNaN(*metric.Value) || math.IsInf(*metric.Value, 0) {
```

Comment:
Do you want to check for being "in the range of -2^360 to 2^360" too?  Telegraf seems to have [something](https://github.com/influxdata/telegraf/blob/67f9a0ea8276e06e3c3d4028cbbd113dfe9bbc6f/plugins/outputs/cloudwatch/cloudwatch.go#L404-L405) around that...

Reply:
I'm not sure what telegraf has, but I can add those bounds. I'm guessing they're inclusive.

Reply:
Having trouble getting the check to validate for values close to the bounds.